liquidctl/driver/nzxt_smart_device.py

```diff
@@ -182,66 +276,239 @@ def get_status(self, **kwargs):
+        'fading':                           (0x01, 0x00, 0x00, 1, 8),
+        'spectrum-wave':                    (0x02, 0x00, 0x00, 0, 0),
+        'backwards-spectrum-wave':          (0x02, 0x00, 0x01, 0, 0),
+        'marquee-3':                        (0x03, 0x00, 0x00, 1, 1),
+        'marquee-4':                        (0x03, 0x01, 0x00, 1, 1),
+        'marquee-5':                        (0x03, 0x02, 0x00, 1, 1),
+        'marquee-6':                        (0x03, 0x03, 0x00, 1, 1),
+        'backwards-marquee-3':              (0x03, 0x00, 0x01, 1, 1),
+        'backwards-marquee-4':              (0x03, 0x01, 0x01, 1, 1),
+        'backwards-marquee-5':              (0x03, 0x02, 0x01, 1, 1),
+        'backwards-marquee-6':              (0x03, 0x03, 0x01, 1, 1),
+        'covering-marquee':                 (0x04, 0x00, 0x00, 1, 8),
+        'covering-backwards-marquee':       (0x04, 0x00, 0x01, 1, 8),
+        'alternating-3':                    (0x05, 0x00, 0x00, 2, 2),
+        'alternating-4':                    (0x05, 0x01, 0x00, 2, 2),
+        'alternating-5':                    (0x05, 0x02, 0x00, 2, 2),
+        'alternating-6':                    (0x05, 0x03, 0x00, 2, 2),    
+        'moving-alternating-3':             (0x05, 0x00, 0x10, 2, 2),   # byte4: 0x10 = moving
+        'moving-alternating-4':             (0x05, 0x01, 0x10, 2, 2),   # byte4: 0x10 = moving
+        'moving-alternating-5':             (0x05, 0x02, 0x10, 2, 2),   # byte4: 0x10 = moving
+        'moving-alternating-6':             (0x05, 0x03, 0x10, 2, 2),   # byte4: 0x10 = moving
+        'backwards-moving-alternating-3':   (0x05, 0x00, 0x11, 2, 2),   # byte4: 0x11 = moving + backwards
+        'backwards-moving-alternating-4':   (0x05, 0x01, 0x11, 2, 2),   # byte4: 0x11 = moving + backwards
+        'backwards-moving-alternating-5':   (0x05, 0x02, 0x11, 2, 2),   # byte4: 0x11 = moving + backwards
+        'backwards-moving-alternating-6':   (0x05, 0x03, 0x11, 2, 2),   # byte4: 0x11 = moving + backwards        
+        'pulse':                            (0x06, 0x00, 0x00, 1, 8),
+        'breathing':                        (0x07, 0x00, 0x00, 1, 8),   # colors for each step
+        'super-breathing':                  (0x03, 0x19, 0x00, 1, 40),  # independent leds
+        'candle':                           (0x08, 0x00, 0x00, 1, 1),
+        'starry-night':                     (0x09, 0x00, 0x00, 1, 1),
+        'rainbow-flow':                     (0x0b, 0x00, 0x00, 0, 0),
+        'super-rainbow':                    (0x0c, 0x00, 0x00, 0, 0),   
+        'rainbow-pulse':                    (0x0d, 0x00, 0x00, 0, 0),
+        'backwards-rainbow-flow':           (0x0b, 0x00, 0x01, 0, 0),
+        'backwards-super-rainbow':          (0x0c, 0x00, 0x01, 0, 0),   
+        'backwards-rainbow-pulse':          (0x0d, 0x00, 0x01, 0, 0),
+        'wings':                            (0xff, 0x00, 0x00, 1, 1),   # wings requires special handling
```

Comment:
Oh, in that case you can use `None` instead of `0xff`, making it clear that it isn't a usable `mval`.

Reply:
Done!